kong/requirements.txt

```diff
@@ -0,0 +1 @@
+# integration pip requirements
```

Comment:
This is a question I've asked myself for other checks as well... Should `simplejson` be here as well? I know it's out in the SDK requirements txt, but I _think_ it might make sense to put these in the respective check's txt as well. What do you think?